service/helpers.py

```diff
@@ -0,0 +1,35 @@
+from datetime import date
+import datetime
+from service.models import DataValidationError
+
+from . import app
+
+def convert_data(data):
```

Comment:
Hi, I saw this code, and from my understanding you are converting the date between two forms (The "yyyy-mm-dd" and timestamp). But I didn't see any place that timestamp exists. Is there any specific reasons that you add these two functions here?

Reply:
The dates are datetime objects within the Promotion object. they come as strings through the REST API. Thus need something to go back and forth. 

Bools are similar.